<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lunaTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="pmLogCtlCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="logCaseCombo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PmLogCtl</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Log State</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addFilterEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addFilterBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="boadrate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Boadrate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COM port</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1341</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>실시간 필터</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1331</width>
        <height>691</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1329</width>
         <height>689</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="logWin">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>1331</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="logStartBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>전체 로그 보기</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1341</width>
        <height>701</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>파일 불러와서 필터</string>
     </attribute>
     <widget class="QPushButton" name="getDirectoryBtn">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="directoryField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>811</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="logWin_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1331</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Timeline으로 보기</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLineEdit" name="comport">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1359</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>856</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pmLogCtlCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>setFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logCaseCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>setFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logStartBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startLogBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendBtnClicked()</slot>
  <slot>setFilter()</slot>
  <slot>startLogBtnClicked()</slot>
  <slot>connectBtnClicked()</slot>
 </slots>
</ui>
